esi_report.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# run ./esi_report.sh daily to get daily report
+# run ./esi_report.sh monthly to get last month's report
+# run ./esi_report.sh weekly to get last week's report
+# run ./esi_reports.sh plan to get future plan
+
+report_dir=/tmp/esi_report
+
+source /home/stack/overcloud-deploy/overcloud/overcloudrc
+
+if [[ $1 == "weekly" ]]; then
```

Comment:
I think we may want to further classify these reports - something like:

```
<root report dir>/
     + leases/
            + current/
                  + <date>-current.csv
            + weekly/
                  + <week>-weekly.csv
            + monthly/
                  + <monthly>-monthly.csv
     + nodes/ 
             + daily/
                  + <date>-daily.log
```
We probably want to rename the options in the script as well then; "weekly" becomes something like "leases-weekly", so we can accommodate future weekly reports of a different type.

Reply:
Fixed!